src/app/src/components/ClaimFacilityConfirmationStep.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+
+import { claimAFacilityFormStyles } from '../util/styles';
+
+export default function ClaimFacilityConfirmationStep() {
+    return (
+        <div style={claimAFacilityFormStyles.inputGroupStyles}>
```

Comment:
Small style issue. Text sometimes runs up against the right side of the wizard panel.

<img width="927" alt="Screen Shot 2019-06-05 at 11 14 11 AM" src="https://user-images.githubusercontent.com/17363/58979577-30297680-8783-11e9-9d6b-789254dac259.png">

Reply:
Fixed this in ee5499c9b4c74eb7c761a672acdab065c021485c
by giving the message's element a width of 95%.

Reply:
![Screen Shot 2019-06-05 at 6 29 41 PM](https://user-images.githubusercontent.com/4165523/58994839-e6ab4c80-87bf-11e9-87f5-080b4a516392.png)

![Screen Shot 2019-06-05 at 6 30 03 PM](https://user-images.githubusercontent.com/4165523/58994850-f4f96880-87bf-11e9-8df1-478333b7171b.png)